nuv/wsk.go

```diff
@@ -48,6 +47,7 @@ func init() {
 
 func Wsk(args ...string) error {
 	os.Args = append([]string{"wsk"}, args...)
+	setWskConfigFile()
```

Comment:
I suggest we do this as BeforeApply hook, see example here
https://github.com/nuvolaris/nuvolaris-cli/blob/e39501d7cb22769a025aa21fa7ef9d1d083b3afa/nuv/deploy.go#L38